BUILD

```diff
@@ -0,0 +1,6 @@
+load("@rules_cc//cc:defs.bzl", "cc_binary")
+
+cc_binary(
+    name = "hello-world",
+    srcs = ["hello-world.cc"],
```

Comment:
Should we use underscores as separators in file names? @asraa is there a specific Envoy convention that we could perhaps follow here?

Reply:
We prefer underscores, but abide by Google style guide which says bother are ok so long as consistent: https://google.github.io/styleguide/cppguide.html#File_Names

Reply:
Vote for underscores. Personally I prefer underscores. The file names here cane from the Bazel document, I thought it's Bazel's tradition and thus copy & paste.

Reply:
Then maybe stick with `_`? I know there are standard directory names for Google projects that follow this convention (e.g., `third_party`).